### PR TITLE
[Flow EVM] use `big.Int` for any balance related fields

### PR DESCRIPTION
**Pull request by ramtinms**
closes: #5222

### CODE REVIEW

File: fvm/evm/types/events.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func (p *BlockExecutedEventPayload) CadenceEvent() (cadence.Event, error) {
 
 	fields := []cadence.Value{
 		cadence.NewUInt64(p.Block.Height),
-		cadence.NewUInt64(p.Block.TotalSupply),
+		cadence.String(p.Block.TotalSupply.String()),
```
Comment (turbolent) [inline fvm/evm/types/events.go:126]: Maybe just use `Int` here?

```suggestion
		cadence.NewIntFromBig(p.Block.TotalSupply),
```

In any case, the type definition for the event, `blockExecutedEventCadenceType`, needs to be adjusted accordingly

Comment (ramtinms) [reply fvm/evm/types/events.go:126]: Nice, didn't know Cadence supports big.Int under the hood for ints larger than the uint64.

Comment (ramtinms) [reply fvm/evm/types/events.go]: updated to big.Int, also fixed the order

**Review by turbolent — APPROVED** [review]
Nice

**Review by janezpodhostnik — APPROVED** [review]
Does it make sense to have a type alias `AttoFlow = *big.Int`

**Review by sideninja — APPROVED** [review]

**Comment by sideninja** [comment]
There's a panic in fvm tests.

**Pull request merged**